eZ/Publish/API/Repository/Values/MultiLanguageValueDescriptionBase.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * File containing the eZ\Publish\API\Repository\Values\MultiLanguageValueBase class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository\Values;
+
+/**
+ * This is the base class for all classes providing translated description logic.
+ *
+ * Provides a uniform way for API consuming logic to generate translated description labels for API objects.
+ */
+abstract class MultiLanguageValueDescriptionBase extends MultiLanguageValueNameBase
```

Comment:
I don't like it.

1. `MultiLanguageValueDescriptionBase` is not a child of `MultiLanguageValueNameBase` - not sure if business logic assumption that description always comes with name is a reason to implement inheritance.

2. Both should be interfaces - their purpose is to ensure method signatures on descendants. Interfaces would also resolve multi-inheritance chaining issue.

```php
abstract class ContentType extends ValueObject implements ValueDescriptionAware, ValueNameAware {}
```

vs. 

```php
abstract class MultiLanguageValueNameBase extends ValueObject {}
abstract class MultiLanguageValueDescriptionBase extends MultiLanguageValueNameBase {}
abstract class ContentType extends MultiLanguageValueDescriptionBase {}
```

3. (ad. 1 + 2) If we need a place where we plan to add basic implementation in a future, maybe implementing base, empty abstract aggregating mentioned interfaces is a way to go.

```php
abstract class MultiLanguageValueBase extends ValueObject implements ValueDescriptionAware, ValueNameAware {}
abstract class ContentType extends MultiLanguageValueBase {}
```